certified-operators/dynatrace/dynatrace-monitoring.0.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,243 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
```

Comment:
A previous PR removed dynatrace from certified, so changes to these files should be removed from your commits.

Reply:
I thought the opposite happened, will fix